libsplinter/src/biome/user/mod.rs

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-mod users;
+//! Users are central entity in the biome module. They represent a real person who uses a splinter
```

Comment:
s/are central/are the central/

Reply:
To avoid the number mismatch ("users = the entity"), I'd suggest: "A user is the central entity in the biome module.  It represents ..."

Also, "Splinter" (cap S).

Reply:
Fixed - This was preexisting text, so I added a commit to fix it instead of updating the existing commit which just moves the text as-is.

Reply:
Fixed 'Splinter'